src/Form/FieldTextarea.js

```diff
@@ -5,30 +5,94 @@ import React from 'react';
 
 import FieldInput from './FieldInput';
 import IconEdit from './icons/IconEdit';
+import Util from '../Util/Util';
+
+class FieldTextarea extends FieldInput {
+  constructor() {
+    super(...arguments);
+
+    this.state = {height: 0};
+
+    this.updateTextareaHeight = Util.throttle(this.updateTextareaHeight, 100);
+    this.handleContentEditableBlur = this.handleContentEditableBlur.bind(this);
+    this.handleContentEditableChange =
+      this.handleContentEditableChange.bind(this);
+    this.handleContentEditableFocus =
+      this.handleContentEditableFocus.bind(this);
```

Comment:
What if we have `FieldInput` implement the BindMixin?
It's [smart](https://github.com/mesosphere/reactjs-components/blob/master/src/Mixin/BindMixin.js#L3) enough to not try to bind things if methodsToBind isn't defined.